CMakeLists.txt

```
@@ -2510,7 +2531,8 @@ endif()
 # Packaging
 set(CPACK_PACKAGE_VENDOR "Mixxx Project")
 set(CPACK_PACKAGE_CONTACT "RJ Skerry-Ryan <rryan@mixxx.org>")
-set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/cmake/cpack_package_description.txt")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Digital Disc Jockey Application")
```

```suggestion
set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Digital DJ Application")
```
I think it is more likely a user would search their package manager for "DJ" than "Disc Jockey".